frontend/src/pages/Overview/OverviewPage.tsx

```diff
@@ -536,13 +536,25 @@ export class OverviewPageComponent extends React.Component<OverviewProps, State>
   async fetchValidationResultForCluster(namespaces: NamespaceInfo[], cluster: string): Promise<void> {
     return Promise.all([API.getConfigValidations(cluster), API.getAllIstioConfigs([], [], false, '', '', cluster)])
```

Comment:
Here a namespaces param is required:
API.getConfigValidations(namespacesToString(namespaces), cluster)